runtime/src/main/java/org/corfudb/runtime/collections/ReadTransaction.java

```diff
@@ -0,0 +1,114 @@
+package org.corfudb.runtime.collections;
+
+import com.google.protobuf.Message;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.Set;
+import java.util.function.BiFunction;
+import java.util.function.BiPredicate;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.stream.Stream;
+
+import static org.corfudb.runtime.collections.QueryOptions.DEFAULT_OPTIONS;
+
+public class ReadTransaction implements ReadTransactionApi {
```

Comment:
Nit: perhaps a comment indicating this class implements the normal Reads of a Read-write Transaction?